build-tools/template/component/{name_sc}/{name_sc}.hbs

```diff
@@ -1,5 +1,5 @@
 <link rel="stylesheet" href="./{{name_sc}}.scss">
 
-<div data-component="{{name_sc}}">
+<div class="{{name_sc}}">
```

Comment:
components should also have both a `data-component` and a `class`

Reply:
Added the data-attribute. I thought the data-component was only used as a reference for typescript.

Reply:
Indeed. The most common error is a mismatch or missing data-component attribute (then the class doesn't get instantiated), so it's best to just keep this in by default (doesn't harm you).